CMakeLists.txt

```diff
@@ -1636,6 +1653,23 @@ add_library_variants_for_cpu(
     RAM_SIZE 0x1000000
     STACK_SIZE 4K
 )
+add_library_variants_for_cpu(
+    armv7r
+    SUFFIX soft_vfpv3_d16
+    COMPILE_FLAGS "-mfloat-abi=soft -march=armv7r -mfpu=vfpv3-d16"
```

Comment:
These options contradict one another - `-mfloat-abi=soft` disables the FPU, but then `-mfpu=vfpv3-d16` enables it. Should this be `-mfloat-abi=softfp` to match the other library variant, or something else?

Reply:
Done